store/mongo/datastore_mongo.go

```diff
@@ -46,6 +47,10 @@ const (
 	KeyTenantID = "tenant_id"
 )
 
+var (
+	ErrPartialUpdate = errors.New("mongo: partial update")
```

Comment:
I cannot find where this variable is used...

Reply:
It was used in the first version of `UpdateConfiguration` 7d2b92b189074ffb87dc88de640340d9af422585. Then I completely changed the implementation in the following commit.
IOW, this variable is unused.
I've squashed out the first version and removed the error.